### PR TITLE
fixed element move command drops instance annotations (SALTO-974)

### PR DESCRIPTION
**Pull request by roironn**

### CODE REVIEW

File: packages/adapter-utils/src/utils.ts [hunk 1]
```diff
@@ -405,6 +405,9 @@ const getPath = (
   const { parent, path } = fullElemID.createTopLevelParentID()
   if (!_.isEqual(parent, rootElement.elemID)) return undefined
   if (_.isEmpty(path)) return []
+  if (isInstanceElement(rootElement) && fullElemID.isAttrID()) {
```
Comment (ori-moisis) [inline packages/adapter-utils/src/utils.ts:408]: why not just `isAttrID` here and remove line 422 that basically does the same thing?

File: packages/adapter-utils/test/utils.test.ts [hunk 2]
```diff
@@ -1078,6 +1078,12 @@ describe('Test utils.ts', () => {
         mockInstance.elemID.createNestedID('str')
       )).toBe('val')
     })
+    it('should resove an instance annotation value path', () => {
```
Comment (ori-moisis) [inline packages/adapter-utils/test/utils.test.ts:1081]:
```suggestion
    it('should resolve an instance annotation value path', () => {
```

File: packages/adapter-utils/test/utils.test.ts [hunk 3]
```diff
@@ -1243,6 +1249,8 @@ describe('Test utils.ts', () => {
     const inst = new InstanceElement('inst', obj, {
       obj: { str: 'Well I do', num: 42 },
       list: ['Do', 'you', 'get', 'it', '?'],
+    }, [], {
+      [INSTANCE_ANNOTATIONS.DEPENDS_ON]: new ObjectType({ elemID: new ElemID('salto', 'dep') }),
```
Comment (ori-moisis) [inline packages/adapter-utils/test/utils.test.ts:1253]: depends on annotation type is a list, value should be a list for this test to be realistic

**Review by ori-moisis — APPROVED** [review]

File: packages/adapter-utils/src/utils.ts (hunk 1 above)
Comment (roironn) [reply packages/adapter-utils/src/utils.ts:408]: 👍

**Pull request merged**